data/src/main/java/com/sincalexandrudaniel/data/feature/news/local/NewsLocalDataStore.java

```diff
@@ -0,0 +1,61 @@
+package com.sincalexandrudaniel.data.feature.news.local;
+
+import android.annotation.SuppressLint;
+import android.util.Log;
+
+import com.sincalexandrudaniel.data.feature.news.local.mapers.ArticleToNewsEntityMapper;
+import com.sincalexandrudaniel.data.feature.news.local.mapers.NewsEntityToArticleMapper;
+import com.sincalexandrudaniel.data.feature.news.model.Article;
+
+import java.util.List;
+
+import io.reactivex.Completable;
+import io.reactivex.Flowable;
+import io.reactivex.Single;
+import io.reactivex.schedulers.Schedulers;
+
+public class NewsLocalDataStore {
+    private static final String TAG = "NewsLocalDataStore";
+
+    private NewsDao newsDao;
```

Comment:
let's make the newsDao final, this way we'll explicitly stay with this object, nothing will change the NewsLocalDataStore: less states, less bugs